backend/pom.xml

```diff
@@ -3,8 +3,8 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.sps</groupId>
-  <artifactId>greyswan</artifactId>
+  <groupId>com.google.backend</groupId>
+  <artifactId>backend</artifactId>
```

Comment:
change artifactId to greySwan (make it specific to our application)

Reply:
sounds good!